pxeboot.py

```diff
@@ -185,15 +187,27 @@ def copy_kickstart(ssh_pubkey: list[str]) -> None:
     kickstart = kickstart.replace("@__DPU_IP4ADDRNET__@", common_dpu.dpu_ip4addrnet)
     kickstart = kickstart.replace("@__HOST_IP4ADDR__@", common_dpu.host_ip4addr)
 
+    res = host.local.run(
```

Comment:
I don't understand why the Marvell DPU will most likely use static IP configuration?

Reply:
The DPU will have DHCP configured and chrony installed/running.

When the DPU (NetworkManager inside the DPU) gets a DHCP lease, then the time servers are picked up from there by chrony, written to `/run/chrony-dhcp/enP2p2s0.sources`. As always.

However, if the DHCP server is stopped (which can happen because you reprovision the host), then chrony will no longer have useful NTP servers (because the public NTP servers that are configured by default, are not reachable).

This prevents that problem.

Chrony anyway will figure out which NTP server to use. This doesn't seem harmful.

Reply:
We can have this in for now. But we should use DHCP to limit the changes to the kickstart file or any customizations to it for hardcoding any names.